packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -444,11 +450,21 @@ contract Deploy is Deployer {
                 _implementation: disputeGameFactory,
                 _data: abi.encodeCall(
                     DisputeGameFactory.initialize,
-                    (cfg.finalSystemOwner())
+                    (tx.origin)
                 )
             });
 
-            string memory version = DisputeGameFactory(disputeGameFactoryProxy).version();
+            DisputeGameFactory factory = DisputeGameFactory(disputeGameFactoryProxy);
```

Comment:
This should be in its own function tbh, if we ever upgrade the dgf we wont need to redeploy the fault vm necessarily